src/Shared/Recruit.Vacancies.Client/Infrastructure/Extensions/VacancyExtensions.cs

```diff
@@ -12,42 +14,41 @@ namespace Esfa.Recruit.Vacancies.Client.Infrastructure.Extensions
 {
     public static class VacancyExtensions
     {
-        public static LiveVacancy ToLiveVacancyProjection(this Vacancy vacancy, ApprenticeshipProgramme programme)
+        public static LiveVacancy ToLiveVacancyProjection(this Vacancy vacancy, ApprenticeshipProgramme programme) => new LiveVacancy
         {
-            return new LiveVacancy
-            {
-                VacancyId = vacancy.Id,
-                ApplicationInstructions = vacancy.ApplicationInstructions,
-                ApplicationMethod = vacancy.ApplicationMethod.Value.ToString(),
-                ApplicationUrl = vacancy.ApplicationUrl,
-                ClosingDate = vacancy.ClosingDate.Value,
-                Description = vacancy.Description,
-                DisabilityConfident = vacancy.DisabilityConfident,
-                EmployerContactEmail = vacancy.EmployerContactEmail,
-                EmployerContactName = vacancy.EmployerContactName,
-                EmployerContactPhone = vacancy.EmployerContactPhone,
-                EmployerDescription = vacancy.EmployerDescription,
-                EmployerLocation = vacancy.EmployerLocation.ToProjection(),
-                EmployerName = vacancy.EmployerName,
-                EmployerWebsiteUrl = vacancy.EmployerWebsiteUrl,
-                LiveDate = vacancy.LiveDate.Value,
-                NumberOfPositions = vacancy.NumberOfPositions.Value,
-                OutcomeDescription = vacancy.OutcomeDescription,
-                ProgrammeId = vacancy.ProgrammeId,
-                ProgrammeLevel = programme.Level.ToString(),
-                ProgrammeType = programme.ApprenticeshipType.ToString(),
-                Qualifications = vacancy.Qualifications.ToProjection(),
-                ShortDescription = vacancy.ShortDescription,
-                Skills = vacancy.Skills,
-                StartDate = vacancy.StartDate.Value,
-                ThingsToConsider = vacancy.ThingsToConsider,
-                Title = vacancy.Title,
-                TrainingDescription = vacancy.TrainingDescription,
-                TrainingProvider = vacancy.TrainingProvider.ToProjection(),
-                VacancyReference = vacancy.VacancyReference.Value,
-                Wage = vacancy.Wage.ToProjection()
-            };
-        }
+            Id = QueryViewType.LiveVacancy.GetIdValue(vacancy.VacancyReference.ToString()),
+            LastUpdated = DateTime.UtcNow,
+            VacancyId = vacancy.Id,
+            ApplicationInstructions = vacancy.ApplicationInstructions,
+            ApplicationMethod = vacancy.ApplicationMethod.Value.ToString(),

```

Comment:
consider changing to ApplicationMethod = vacancy.ApplicationMethod?.ToString() in case of nulls
there are a few of these in the mapping methods.